pkgs/development/compilers/ponyc/make-safe-for-sandbox.patch

```diff
@@ -0,0 +1,93 @@
+--- a/lib/CMakeLists.txt	2020-09-27 02:39:12.862940178 +0000
++++ b/lib/CMakeLists.txt	2020-09-27 02:39:16.451957865 +0000
+@@ -10,12 +10,12 @@
+ endif()
+ 
+ ExternalProject_Add(gbenchmark
+-    URL https://github.com/google/benchmark/archive/v1.5.0.tar.gz
++    SOURCE_DIR gbenchmark-prefix/src/benchmark
+     CMAKE_ARGS -DCMAKE_BUILD_TYPE=${PONYC_LIBS_BUILD_TYPE} -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX} -DBENCHMARK_ENABLE_GTEST_TESTS=OFF -DCMAKE_CXX_FLAGS=-fpic --no-warn-unused-cli
+ )
+ 
+ ExternalProject_Add(googletest
+-    URL https://github.com/google/googletest/archive/release-1.8.1.tar.gz
++    URL /nix/store/pasaddah8gabrvws74lvr9szc4l82hw9-release-1.8.1.tar.gz
```

Comment:
This will break on systems where store is not located at /nix/store. You might want to do
```suggestion
+    URL @googletest@
```
and then use
```nix
substituteAll {
  src = ./make-safe-for-sandbox.patch;
  googletest = fetchurl {
    url = https://github.com/google/googletest/archive/release-1.8.1.tar.gz;
    sha256 = "17147961i01fl099ygxjx4asvjanwdd446nwbq9v8156h98zxwcv";
    name = "release-1.8.1.tar.gz";
  };
}
```

Reply:
Done!